src/index.js

```diff
@@ -0,0 +1,51 @@
+import _ from 'lodash';
+import './style.css';
+import * as el from './modules/elementSelector.js';
+import generateTaskList from './modules/generateTaskList.js';
+
+// import { draggables } from './modules/elementSelector';
```

Comment:
- [OPTIONAL] It will be nice to remove commented code from your code base, as this makes your code base dirty and less readable.